projects/vdk-core/src/vdk/api/plugin/plugin_input.py

```diff
@@ -387,6 +387,7 @@ def pre_ingest_process(
         target: Optional[str] = None,
         collection_id: Optional[str] = None,
         metadata: Optional[IngestionMetadata] = None,
+        method: Optional[str] = None,
```

Comment:
Please add this to the post_ingest_process. 

They should be consistent.

Reply:
Done. Thanks